setup/macros_scoresheets.tex

```diff
@@ -560,7 +560,8 @@
 
 	% require signal for door opening %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 	\ifthenelse{ \equal{\scorelistStartButton}{true} }{
-	  \penaltyitem{\scorelistStartButtonPenalty}{Using alternative start signal \ifShortScoresheet{(see sec.~\ref{rule:start_signal})}{}}
+	  \penaltyitem{\scorelistStartButtonPenalty}{Using alternative start signal %\ifShortScoresheet{(see sec.~\ref{rule:start_signal})}
```

Comment:
Again remove it if not needed. But why does this need to go?

Reply:
Start_signal is part of the procedure section which is now in the orga doc.

We have to reorganise both documents but the rulebook is already released so I just want to fix the missing reference for now.

Alternatively we could insert `see section xy of organisation document`

Reply:
We could also move the section of start signal and gestures to the rulebook since these have more to do with how tests are run. 
Then we would have the same problem with refering to the RIPS, but I think we can fix it by saying 
```
see \Organisation
```